xbmc/utils/StreamUtils.cpp

```diff
@@ -150,3 +150,9 @@ std::string StreamUtils::GetLayout(unsigned int channels)
 
   return layout;
 }
+
+bool StreamUtils::IsCodecSupportForcedOverlay(int codecId)
+{
+  return codecId == AV_CODEC_ID_DVD_SUBTITLE || codecId == AV_CODEC_ID_HDMV_PGS_SUBTITLE ||
+         codecId == AV_CODEC_ID_DVB_SUBTITLE;
```

Comment:
`AV_CODEC_ID_DVB_SUBTITLE` is listed twice?

Reply:
You're right. I even tried comparing pixels and missed the difference. Not sure if code can be made more clear but it's correct as is.